cyber/examples/BUILD

```diff
@@ -21,6 +22,19 @@ cc_binary(
     ],
 )
 
+install(
+    name = "install",
+    targets = [
+        ":talker",
+        ":listener",
+    ],
+    runtime_dest = "cyber/examples",
+    deps = [
```

Comment:
Install the `talker` and `listener` demo programs to the `cyber/examples` directory rather than `bin`